src/main/mulmo/handler.ts

```
@@ -275,7 +276,9 @@
 const mediaFilePath = async (projectId: string, actionName: string) => {
   const context = await getContext(projectId);
   if (actionName === "audio") {
-    return audioFilePath(context);
+    console.log(getAudioArtifactFilePath(context))
```

ESLint

[error] Insert `;`

(prettier/prettier)